mlx/directives/item_attributes_matrix_directive.py

```diff
@@ -155,21 +155,11 @@ def run(self):
 
         # Process ``attributes`` option, given as a string with attributes
         # separated by space. It is converted to a list.
-        if 'attributes' in self.options and self.options['attributes']:
-            self._warn_if_comma_separated('attributes', env.docname)
-            node['attributes'] = self.options['attributes'].split()
-        else:
-            node['attributes'] = list(app.config.traceability_attributes)
-        self.remove_unknown_attributes(node['attributes'], 'attribute', env.docname)
```

Comment:
this line was executed at default state, but with new one it is not.  However at sort it was not, but I do not know if we did not detect duplicates or at this point in time there already are no duplicates. I would rather remove duplicates twice than none at all...

Reply:
I verified that this check is not needed when `default` is `app.config.traceability_attributes` or `[]`, but since this is a public method, it *should* call `self.remove_unknown_attributes` when `default` is used, just to be safe.  aac88c2ea8640c87c78e860029f6b4863df62635